File: frontend/SetupMeeting/src/setupmeeting/model/meeting.ts
export class Meeting{
    meetingId:string;
    name:string;
    startAt:Date;
    endAt:Date;
    organizers:OrganiZer[];
    invitees:Invite[];
    attendees:Attende[];
    sponsers:OrganiZer[];
    status:MeetingStatus =0;

    constructor(){
        this.organizers = [];
        this.invitees = [];
        this.attendees = [];
        this.sponsers = [];
    }
}

export class OrganiZer{
    userId:string;
    userName:string;
    organizationId:string;
    organizationName:string;
}

export class Invite{
    userId:string;
    userName:string;
    organizationId:string;
    organizationName:string;
    rsvp:boolean;

    constructor(){
        this.rsvp = false;
    }
}

export class Attende{
    userId:string;
    userName:string;
    organizationId:string;
    organizationName:string;
    attend:boolean;
    payed:boolean;

    constructor(){
        this.attend = false;
    }
}


export enum MeetingStatus{
    Before=0,
    Open=1    
}

File: backend/src/SetupMeetings.Queries/Meetings/Meeting.cs
using System;
using System.Collections.Generic;

namespace SetupMeetings.Queries.Meetings
{
    public class Meeting
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public DateTime StartAt { get; set; }
        public DateTime EndAt { get; set; }
        public List<Organizer> Organizers { get; set; } = new List<Organizer>();
        public List<Attendee> Attendees { get; set; } = new List<Attendee>();
        public List<Invitee> Invitees { get; set; } = new List<Invitee>();
        public List<Sponsor> Sponsors { get; set; } = new List<Sponsor>();
    }
}